.circleci/config.yml

```diff
@@ -255,6 +256,7 @@ jobs:
           name: Repeatability of Data -> Target
           command: |
             source venv--target-postgres/bin/activate
+            pip install .
```

Comment:
unsure as to _why_ this is needed but somehow the `virtualenv` ended up getting borked somewhere along the way.